src-electron/db/db-mapping.js

```diff
@@ -610,7 +610,7 @@ exports.map = {
       sessionId: x.SESSION_ID,
       sessionKey: x.SESSION_KEY,
       creationTime: x.CREATION_TIME,
-      packageRef: x.PACKAGE_REF,
```

Comment:
Can this cause backwards compatibility issues? We should not be changing the mapping in general. Keep the existing ones for backwards compatibility.